### ISSUE TITLE
Heard list on http

### ISSUE THREAD

**Issue by cminyard** [la5nta/pat#337]
I don't see a way to see the "heard" list when using the http interface.  That would be nice.

**Comment by martinhpedersen** [comment]
Agreed! I'm very happy with the heard list, and would love to see support for it in the web gui 👍

**Comment by JoshuaCarroll** [comment]
What is the heard list?

**Comment by xylo04** [comment]
If you have your radio+modem listening for incoming connections, it can do more than listen just for its own callsign; it can report every callsign that it has heard on the air, and thus give you an idea of which stations might be able to hear you if you were to call them. I haven't actually used it but I take it this is a feature already in the CLI.
